content/blocksPages/home.json

```
@@ -185,7 +185,7 @@
         },
         {
           "headline": "3rd Party Media",
-          "text": "Store Media assets in your repo or connect to S3, Cloudinary, DigitalOcean Spaces, etc. Or, use TinaCloud",
+          "text": "Store Media assets is your repo or connect to S3, Cloudinary, DigitalOcean Spaces, etc. Or, use TinaCloud",
```

```suggestion
          "text": "Store Media assets in your repo or connect to S3, Cloudinary, DigitalOcean Spaces, etc. Or, use TinaCloud",
```
Not sure why this text was changed. The change lowers readability